bitnami/grafana-operator/templates/deployment.yaml

```diff
@@ -120,10 +120,10 @@ spec:
             - name: TEMPLATE_PATH
               value: /usr/local/bin/templates
             - name: WATCH_NAMESPACE
-              {{- if and .Values.namespaceScope (empty .Values.watchNamespaces) }}
-              value: namespace: {{ include "common.names.namespace" . | quote }}
-              {{ else }}
-              value: {{ .Values.watchNamespaces }}
+              {{- if .Values.namespaceScope }}
+              value: {{ default (include "common.names.namespace" .) .Values.watchNamespace | quote }}
```

Comment:
Is the removal of `namespace: ` expected?

Reply:
Expected and required